### PR TITLE
Add okta IDP configs

### PR DESCRIPTION
**Pull request by deepankermishra**

### CODE REVIEW

File: charts/dynamoai-base/keycloak-config/realm.json [hunk 1]
```diff
@@ -44,6 +44,504 @@
...
+  "groups": [
+    {
+      "name": "all-users",
+      "path": "/all-users",
+      "subGroups": [],
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {}
+    }
+  ],
```
Comment (ishucr7) [inline charts/dynamoai-base/keycloak-config/realm.json:544]: We don't need this yet

File: charts/dynamoai-base/keycloak-config/realm.json [hunk 2]
```diff
@@ -98,6 +596,10 @@
       "totp": false,
       "serviceAccountClientId": "api",
       "disableableCredentialTypes": [],
+      "realmRoles": ["uma_authorization"],
+      "clientRoles": {
+        "realm-management": ["realm-admin"]
+      },
```
Comment (ishucr7) [inline charts/dynamoai-base/keycloak-config/realm.json:602]: Same, no need for this yet!

File: charts/dynamoai-base/keycloak-config/realm.json [hunk 3]
```diff
@@ -354,7 +859,228 @@
...
+            "displayName": "UserGet"
+          },
+          {
+            "name": "user:setRole",
+            "iconUri": "",
+            "displayName": "UserSetRole"
+          }
+        ],
+        "decisionStrategy": "AFFIRMATIVE"
+      }
```
Comment (ishucr7) [inline charts/dynamoai-base/keycloak-config/realm.json:1083]: No need for this too!

**Review by ishucr7 — APPROVED** [review]
LGTM, left a few comments

**Review by brarkaran — APPROVED** [review]

**Pull request merged**